platform.md

```diff
@@ -594,8 +599,14 @@ Usage:
 
 - The lifecycle SHALL write a [report](#reporttoml-toml) to `<report>` describing the exported app image
 
+- The `<layers>` directory:
+  - MUST include all buildpack-provided standardized Bill of Materials (sBOM) files for `build` as determined by the [Buildpack Interface Specfication](buildpack.md)
```

Comment:
Shouldn't this be a **MAY** given that they are optional?

```suggestion
  - MAY include all buildpack-provided standardized Bill of Materials (sBOM) files for `build` as determined by the [Buildpack Interface Specfication](buildpack.md)
```

Reply:
Same comment as https://github.com/buildpacks/spec/pull/268/files#r746701237

Reply:
Clarified in cf67713